deploy/helm/elastic-agent/README.md

```diff
@@ -144,8 +144,8 @@ The chart built-in [kubernetes integration](https://docs.elastic.co/integrations
 ### 6 - Elastic-Agent Configuration
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
-| agent.version | string | `"9.0.0"` | elastic-agent version |
-| agent.image | object | `{"pullPolicy":"IfNotPresent","repository":"docker.elastic.co/elastic-agent/elastic-agent","tag":"9.0.0-SNAPSHOT"}` | image configuration |
+| agent.version | string | `"9.1.0"` | elastic-agent version |
```

Comment:
We should also auto-generate this table. Should be relatively straightforward from the schema file.

Reply:
that's a fair point! ty for raising it, created [this issue](https://github.com/elastic/elastic-agent/issues/7489) 🙂